src/client/theme-default/components/VPLocalNavOutlineDropdown.vue

```diff
@@ -155,11 +161,12 @@ function scrollToTop() {
 
 .header {
   background-color: var(--vp-c-bg-soft);
+  padding: 2px 16px;
 }
 
 .top-link {
   display: block;
-  padding: 0 16px;
+  padding: 0;
   line-height: 48px;
```

Comment:
```suggestion
  line-height: 44px;
```

Reply:
I just added the div container to allow inert work...

Reply:
upps, it is about the outline

Reply:
Yeah that div thing on skip link is fine.